packages/vuetify/src/composables/theme.ts

```diff
@@ -82,13 +82,14 @@ const defaultThemeOptions: ThemeOptions = {
     light: {
       dark: false,
       colors: {
-        background: '#e5e5e5',
-        surface: '#aaaaaa',
-        primary: '#6200ee',
+        background: '#FFFFFF',
+        border: '#e0e0e0',
```

Comment:
Adding border color here creates classes and variables that don't make much sense in context, like `.bg-border` and `.text-border`, `v-theme-on-border` etc.

I'm not sure exactly how you want outlined to work, but we already generate `.border-x` classes, which could be used. So `outlined="primary"` would apply `.border-primary` class.

I get that you want a specific 'default' border color, but if so we maybe need to rethink the theme class generation a bit. Or maybe use background/surface color with an opacity?